janus_core/calculations/phonons.py

```diff
@@ -262,9 +262,9 @@ def __init__(
         self.displacement = displacement
         self.displacement_kwargs = displacement_kwargs
         self.mesh = mesh
-        self.symmetrize = symmetrize
```

Comment:
For alphabetical ordering?

Reply:
In general I try to set the attributes in the order the parameters are passed where possible. In this case, that should be before minimise and minimise_kwargs, so not completely sure what happened...

At some point I need to do a pass on Python and CLI parameter orderings anyway to bring more consistency again anyway, so I'll fix this properly then.